tools/check-fenced-code.mjs

```diff
@@ -0,0 +1,65 @@
+import { promisify } from 'util';
+import fs from 'fs-extra';
+import g from 'glob';
+import MarkdownIt from 'markdown-it';
+import shell from 'shelljs';
+
+const glob = promisify(g);
+
+const errorTitle = 'Invalid JSON in fenced code block';
+const errorBody =
+  'Fix this manually by ensuring each block is a valid, complete JSON document';
+const markdownGlob = 'docs/**/*.md';
+const markdown = new MarkdownIt('zero');
+
+let issues = 0;
+
+markdown.enable(['fence']);
+
+function checkValidJson(file, token) {
+  const start = parseInt(token.map[0], 10) + 1;
+  const end = parseInt(token.map[1], 10) + 1;
+
+  try {
+    JSON.parse(token.content);
+  } catch (err) {
+    issues += 1;
+    if (process.env.CI) {
+      shell.echo(
+        `::error file=${file},line=${start},endLine=${end},title=${errorTitle}::${err.message}. ${errorBody}`
+      );
+    } else {
+      shell.echo(
+        `${errorTitle} (${file} lines ${start}-${end}): ${err.message}`
+      );
+    }
+  }
+}
+
+async function processFile(file) {
+  const text = await fs.readFile(file, 'utf8');
+  const tokens = markdown.parse(text, undefined);
+  shell.echo(`Linting ${file}..`);
```

Comment:
```suggestion
  shell.echo(`Linting ${file}...`);
```

I think three dots looks nicer for in-progress stuff, two dots looks like a typo to me. 😄

Reply:
Ah I see it's like that in other files too, done!